File: sdkwork-backend-sdk-go/types/plus_news_vo.go
package types

// 新闻资讯VO对象
type PlusNewsVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id int `json:"id"`
	Uuid string `json:"uuid"`
	UserId int `json:"userId"`
	Title string `json:"title"`
	Url string `json:"url"`
	Source string `json:"source"`
	Summary string `json:"summary"`
	PublishTime string `json:"publishTime"`
	Tags TagsContent `json:"tags"`
}

File: sdkwork-backend-sdk-csharp/Models/PlusNewsVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PlusNewsVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public int? Id { get; set; }
        public string? Uuid { get; set; }
        public int? UserId { get; set; }
        public string? Title { get; set; }
        public string? Url { get; set; }
        public string? Source { get; set; }
        public string? Summary { get; set; }
        public string? PublishTime { get; set; }
        public TagsContent? Tags { get; set; }
    }
}
